.gitignore

```diff
@@ -9,3 +9,7 @@ log.html
 .env
 .DS_Store
 /.build
+
+a.out.js
+a.out.wasm
+hcl.so
```

Comment:
are these artifacts generated with `tree-sitter` somehow? I feel like you shouldn't need these here but correct me if I'm wrong

Reply:
Yeah if I test with --wasm flag they get generated I think

Reply:
Note that the output is platform dependent -- you want to exclude `*.so`, `*.dll` and `*.dylib` (and `*.wasm`) instead.

`a.out` should not happen; that looks like a broken binding (makefile) somewhere in this repo.

Reply:
let me revert this file real quick!